trin-state/src/utils.rs

```diff
@@ -50,6 +50,8 @@ impl ToU256 for BigInt {
 // distance function as defined at...
 // https://notes.ethereum.org/h58LZcqqRRuarxx4etOnGQ#Storage-Layout
 // todo: measure performance & optimize if necessary
+// naked wraps ok here because we're operating on static values
+#[allow(clippy::unwrap_used)]
```

Comment:
Slight preference to move this down to the `unwrap()` usages, rather than allowing the whole method.

Also, between the overhead of adding allow, and the comment, I am thinking it might be better to generally prefer `expect`s, even when doing static parsing. Like:

```rs
   let modulo = BigInt::parse_bytes(&MODULO, 10).expect("Parse static MODULO always works");
   let mid = BigInt::parse_bytes(&MID, 10).expect("Parse static MID always works");
```

This kind of thing could even be added to the style guide as a recommendation.